tests/RPCServer.test.ts

```diff
@@ -1016,7 +1016,86 @@ describe(`${RPCServer.name}`, () => {
 
       // Longer timeout is set to server's default
       const ctxLong = await ctxLongProm.p;
-      expect(ctxLong.timer.delay).toEqual(50);
+      expect(ctxLong.timer.delay).toEqual(26);
+      waitProm.resolveP();
+      await rpcServer.stop({ force: true });
+    }
+  });
+  test('handler overrides timeout - greater value', async () => {
+    {
+      const waitProm = promise();
+      const ctxShortProm = promise<ContextTimed>();
+      class TestMethodShortTimeout extends UnaryHandler {
+        handlerTimeout = 250;
+        public handle = async (
+          input: JSONValue,
+          _cancel,
+          _meta,
+          ctx_,
+        ): Promise<JSONValue> => {
+          ctxShortProm.resolveP(ctx_);
+          await waitProm.p;
+          return input;
+        };
+      }
+      const ctxLongProm = promise<ContextTimed>();
+      class TestMethodLongTimeout extends UnaryHandler {
+        handlerTimeout = 261;
+        public handle = async (
+          input: JSONValue,
+          _cancel,
+          _meta,
+          ctx_,
+        ): Promise<JSONValue> => {
+          ctxLongProm.resolveP(ctx_);
+          await waitProm.p;
+          return input;
+        };
+      }
+      const rpcServer = new RPCServer({
+        handlerTimeoutTime: 50,
```

Comment:
I'm confused. I thought the priority order is:

Call timeout overrides default caller timeout.

Handler timeout overrides default handler timeout.

Where are these explicitly overridden?

Reply:
Was this question answered? why is this resolved?